grpc-zpages/docker/start_docker.sh

```diff
@@ -12,7 +12,7 @@ fi
 
 readonly DOCKER_DIR="$(cd "$(dirname "$0")" && pwd)"
 
-TEMP=$(mktemp -p $DOCKER_DIR --suffix=.yaml)
+TEMP=$(mktemp $DOCKER_DIR/temp.yaml)
```

Comment:
It looks like `mktemp $DOCKER_DIR/temp.yaml` is giving me issues on linux. I think we can get around this osx/linux issue by changing the script to read from stdin

```bash
CONFIG="version: '2'
services:
  front-envoy:
    build:
      context: ./envoy/
      dockerfile: Dockerfile
    ports:
      # Forward host port to docker
      - '$ENVOY_PORT:$ENVOY_PORT'
    environment:
      - GRPC_HOST=$GRPC_ADDR
      - GRPC_PORT=$GRPC_PORT
      # The docker port for envoy
      - ENVOY_PORT=$ENVOY_PORT
      # The host+port for the static assets server
      - ASSETS_HOST=127.0.0.1
      - ASSETS_PORT=$ASSETS_PORT
    mem_limit: 1000000000
    network_mode: 'host'
  back-static-assets:
    build:
      context: ./static-assets/
      dockerfile: Dockerfile
    environment:
      - PORT=$ASSETS_PORT
    ports:
      # Make the port availble to linked dockers but no need
      # to forward to host machine.
      - '$ASSETS_PORT:$ASSETS_PORT'
    mem_limit: 1000000000"

cd $DOCKER_DIR
echo "$CONFIG" | docker-compose -f - up --build 
```